src/main/java/ai/rev/StreamContentType.java

```diff
@@ -0,0 +1,170 @@
+package ai.rev;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The StreamContentType describes the format of the audio being sent over the WebSocket.
+ *
+ * @see <a
+ *     href="https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Content-Type">https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Content-Type</a>
+ */
+public class StreamContentType {
+
+  private String contentType;
+  private String layout;
+  private Integer rate;
+  private String format;
+  private Integer channels;
+
+  public StreamContentType() {
+    this.contentType = null;
```

Comment:
minor, but I dont think these initializations are necessary, null is the default value.